<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MazeCave</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>191</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mazeTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="loadFileButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load File</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>169</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="SizeSpinBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="GenerateMazeButton">
         <property name="text">
          <string>GenerateMaze</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="saveMazeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Maze</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="caveTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="randomCaveButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Random Cave</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="coordLayout">
       <item row="1" column="1">
        <widget class="QSpinBox" name="caveYspnBox">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="caveYLavel">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="caveXLabel">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="caveXspnBox">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>189</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="limitsLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="birthLabel">
         <property name="text">
          <string>birth limit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="birthSpinBox">
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="deathLabel">
         <property name="text">
          <string>death limit</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="deathSpinBox">
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="chanceLayout">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="chanceSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="chanceLabel">
         <property name="text">
          <string>Chance of Init</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="nextStepButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Render Next Step</string>
      </property>
     </widget>
     <widget class="QPushButton" name="getCavePathButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Cave</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveCaveButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Cave</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AutoEvolvButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Evolution</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QSpinBox" name="timeSpinBox">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>milliseconds</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="baseGV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>502</width>
      <height>502</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
